### ISSUE TITLE
Non-fullscreen route

### ISSUE THREAD

**Issue by jamesfzhang** [aksonov/react-native-router-flux#165]
I want to add a "flash message" route so that it can be initiated from all other screens. Ideally, this flash message component should only be 80 pixels in height, and sits on top of the current route so that the current route is visible behind the flash component.

Is there a way to do this? I tried using the "modal" schema and setting a fixed height for the flash message component but that still creates a full-screen route.

**Comment by aksonov** [comment]
Have you tried type='modal' ? Check Example, Error popup is definitely not full screen...

**Comment by jamesfzhang** [comment]
I said I did use "modal" but I guess I was on an older version of this library. Is there a change log? The [release page](https://github.com/aksonov/react-native-router-flux/releases) still says 2.0.2 is a pre-release?

**Comment by aksonov** [comment]
Why don't use latest release from npm? Anyway, i've updated release page with latest release.

**Comment by jamesfzhang** [comment]
I was using an earlier version due to this specific behavior I wanted. I just updated to the latest version and implemented that behavior: https://github.com/jamesfzhang/react-native-router-flux/commit/be18b3583fb15be91c213d12f0c8f1e64fda49fa

Essentially, I wanted "switch" behavior for "push" routes since I encountered a bug where I couldn't hide the navigation bar for "switch" routes. I also added an ability to reset a particular route ad-hoc.

Anyway, I updated to the latest version by my modal is still full screen. I'll check out your examples and see what the differences are.
